cg/meta/workflow/analysis.py

```diff
@@ -137,6 +137,12 @@ def get_slurm_qos_for_case(self, case_id: str) -> str:
         priority: int = case.priority or Priority.research
         return Priority.priority_to_slurm_qos().get(priority)
 
+    def get_trailblazer_priority(self, case_id: str) -> int:
+        """Get the priority for the case in Trailblazer."""
+        case: Case = self.status_db.get_case_by_internal_id(internal_id=case_id)
+        priority: int = case.priority or Priority.research
```

Comment:
If `case.priority` is an int, then we are basically saying priority = case.priority _unless it is 0, in which instance we should set it to 0_. I think using the case.priority in all cases should work?

Reply:
In the order portal I checked that a case is required to have a priority so I think I can remove it 👍